packages/devextreme/js/__internal/ui/chat/chat.ts

```diff
@@ -136,6 +139,9 @@ class Chat extends Widget<Properties> {
       items = [],
       user,
       showDayHeaders = true,
+      showAvatar = true,
+      showUserName = true,
+      showMessageTimestamp = true,
```

Comment:
[corner case]
These `= true` are used only in case when we intentionally pass `undefined` the the Chat's option.
And for this case it looks strange to me that we gonna handle it as `true` because undefined is usually handled as `false`. Moreover, I'm pretty sure that it'll be handled as `false` now it we do it at runtime through `.option(optionName, undefined)`.
So, I suggest you using `false` here and add tests for this.
P.s. Kindly update for `showDayHeaders` as well

Reply:
Should I also test the case when we pass undefineed on init?

Reply:
Altough it works as undefined is falsy, the value itself will be undefined.

Reply:
3db30f88525eb513cd10b51d925665c3c981bdf2